File: app/model/Post.js
con = require("../../config/connection");


module.exports = {
    CreatePost : (Post , callback )=>{
        con.query("INSERT INTO phone_indicator.post" +
        " (post_description , post_img ,store_id) VALUES (?,?,?)" , 
        [Post.post_description , Post.post_img , Post.store_id],callback)
    },
    DeletePost : (id ,callback)=>{
        con.query("DELETE FROM phone_indicator.post WHERE post_id =?" , [id] , callback)
    },
    UpdatePost : (Post ,callback)=>{
        con.query("UPDATE phone_indicator.post " +
        " SET post_description = ?, post_img = ? WHERE  store_id = ?" , 
        [Post.post_description , Post.post_img , Post.store_id],callback)
    },
    findById : (id ,callback)=>{
        con.query("SELECT * FROM phone_indicator.post WHERE post_id =?" , [id] , callback)
    },
    findByStoreId : (store_id ,callback)=>{
        con.query("SELECT * FROM phone_indicator.post WHERE store_id =?" , [store_id] , callback)
    },
    findAll : (callback)=>{
        con.query("SELECT * FROM phone_indicator.post" , callback)
    }
    
}

File: project/models/Post.py
from project.config.Database import connection
from flask import jsonify


class Post:

    def __init__(self):
        self.conn = connection()
        self.cursor = self.conn.connection.cursor(dictionary=True)

    def add_post(self, params):
        self.cursor.execute("INSERT INTO mobilaty.post" +
                            " (post_description , post_img ,store_id)" +
                            " VALUES ('" + params['post_description'] + "','" + params['post_img'] +
                            "','" + params['store_id'] + "')")
        self.conn.connection.commit()

    def delete_post(self, params):
        self.cursor.execute('DELETE FROM  mobilaty.post WHERE post_id= ' + params['post_id'])
        self.conn.connection.commit()
        # row = cursor.fetchone()

    def show_AllPost(self):
        self.cursor.execute('SELECT * FROM mobilaty.post')
        row = self.cursor.fetchall()
        return row

    def update_post(self, params):
        self.cursor.execute("update mobilaty.post set post_description = '" +
                            params['post_description'] + "'and post_img= '" +
                            params['post_img'] + "'   where post_id = " + params['post_id'])
        self.conn.connection.commit()

    def show_post(self, params):
        self.cursor.execute('SELECT * FROM mobilaty.post WHERE post_id = ' + params['post_id'] + " ORDER BY RAND()")
        row = self.cursor.fetchall()
        return row

    def findByStoreId(self, params):
        self.cursor.execute('SELECT * FROM mobilaty.post WHERE store_id = ' + params['store_id'])
        row = self.cursor.fetchall()
        return row
